packages/story-editor/src/components/videoTrim/provider.js

```diff
@@ -40,6 +48,40 @@ function VideoTrimProvider({ children }) {
     resetOffsets,
   } = useVideoNode();
 
+  /**
+   * Converts milliseconds to HH:MM:SS.SSS format.
+   *
+   * @type {function(*): string}
+   */
+  const msToHMS = useCallback((ms) => {
```

Comment:
We have some similar function already in the media package. See `getVideoLength` and `formatDuration`

Haven't looked closely, but perhaps it's possible to reuse something there by extracting into a shared function or something.

Reply:
Looks like the logic in `getVideoLength` is a bit different and is using full seconds. Added a new util using `formatDuration` and added it to the media package.